packages/styling/src/styles/theme.ts

```diff
@@ -89,6 +89,23 @@ export function loadTheme(theme: IPartialTheme, depComments: boolean = false): I
   return _theme;
 }
 
+/**
+ * Loads font variables into a JSON object.
+ * @param theme - The theme object
+ */
+export function loadFonts(theme: ITheme): { [name: string]: string } {
```

Comment:
We definitely want to limit unneeded exports. Otherwise, as the api jsons illustrate, they become a public contract. We want to reduce that api surface.

Reply:
I'd personally rather duplicate it in the build script than add more api surface. It's not a large function.

Reply:
It's possible you could separate it out into its own file, export it in that file but *not* through any top level import files, import it from this file as well as another project which generates the scss files. Then it doesn't get exported through any library but can be reused.

Reply:
Okay, cool. I think I'll just make it a private helper for now.

Reply:
In the next PR we can see what we think about the placement of the script to generate the scss files and figure out whether duplicate code or importing this function from a file in another project makes more sense.